modules/storage.js

```diff
@@ -1,10 +1,36 @@
-function createScores(scores) {
-  scores = localStorage.getItem('scores') ? JSON.parse(localStorage.getItem('scores')) : [];
-  return scores;
+async function createScores() {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Z3iGcEUE1HlerNd8QYix/scores/', {
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+
+  if (!response.ok) {
+    throw new Error('Error al obtener los puntajes');
+  }
+
+  const json = await response.json();
+  return json;
 }
```

Comment:
- [ ] Ouch! you've done everything to perfection but you seem to be missing one of the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md) 😟 that states that you should `arrow functions` instead of the `function` keyword. So please make sure to refactor your code accordingly.
- [ ] This applies to all functions in your project declared with the `function` keyword.